pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -278,14 +278,20 @@ func (a *AgentClusterInstall) Load(f asset.FileFetcher) (bool, error) {
 		agentClusterInstall.Spec.PlatformType = hiveext.BareMetalPlatformType
 	case none.Name:
 		agentClusterInstall.Spec.PlatformType = hiveext.NonePlatformType
-		if agentClusterInstall.Spec.Networking.UserManagedNetworking != swag.Bool(true) {
-			logrus.Debugf("Setting UserManagedNetworking to true for %s platform", none.Name)
-			agentClusterInstall.Spec.Networking.UserManagedNetworking = swag.Bool(true)
-		}
 	case vsphere.Name:
 		agentClusterInstall.Spec.PlatformType = hiveext.VSpherePlatformType
 	}
 
+	// Set the default value for userManagedNetworking, as would be done by the
+	// mutating webhook in ZTP.
+	if agentClusterInstall.Spec.Networking.UserManagedNetworking == nil {
+		switch agentClusterInstall.Spec.PlatformType {
+		case hiveext.NonePlatformType:
```

Comment:
The linter is complaining about this being a switch statement instead of an 'if'. The switch statement will make sense when adding the `external` platform here, so maybe disable the lint check.

Reply:
Yes, this now also needs to be set in case of `external` platform